### ISSUE TITLE
Add th cells to the table on the a11y-automation.dev/violations page

### ISSUE THREAD

**Issue by RichCaloggero** [MelSumner/a11y-automation#371]
## Description

When I browse the table with my screen reader, as I move across columns all I hear is the data in the cell; table column headers are not read.

## To reproduce

- go to https://a11y-automation.dev/violations
- start your screen reader
- navigate to the table
- use screen reader commands to navigate to a table cell
- only cell data is read

## Expected behavior

cell data, prefixed by the contents of the corresponding column's data cell should be announced by the screen reader

## Using

- win10
- Firefox
- NVDA screen reader

**Comment by MelSumner** [comment]
Thank you for reporting this issue! I'll get it fixed!

**Comment by MelSumner** [comment]
Ok so I've dug into this some more, and I'm unable to reproduce this issue. The accessibility tree is showing the expected information. 

Can you try turning on your screen reader first, and then opening a browser, and then navigating to the page? I'll dig some more as well and see if I can identify the issue.

**Comment by patrickhlauke** [comment]
I'm assuming the request here is to have the first cell in each row marked up as `<th scope="row" ...>` rather than just `<td>` as is currently the case. This way, jumping from row to row in a column, the relevant row header cell will be announced. And I'd say that'd be a good change, yes.

**Comment by MelSumner** [comment]
Hi @patrickhlauke! Thanks for that feedback. Happy to implement!

**Comment by MelSumner** [comment]
I've implemented the changes, so I'll close this issue for now. Please let me know if it doesn't meet your needs or if further tweaks are needed. Thank you!
